core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/SimplifyExpressions.java

```diff
@@ -39,7 +39,7 @@
         extends ExpressionRewriteRuleSet
 {
     @VisibleForTesting
-    static Expression rewrite(Expression expression, Session session, SymbolAllocator symbolAllocator, Metadata metadata, LiteralEncoder literalEncoder, TypeAnalyzer typeAnalyzer)
+    public static Expression rewrite(Expression expression, Session session, SymbolAllocator symbolAllocator, Metadata metadata, LiteralEncoder literalEncoder, TypeAnalyzer typeAnalyzer)
```

Comment:
Minor nit: remove @VisibleForTesting annotation since it's used outside of testing now.